rasa/core/brokers/pika.py

```diff
@@ -138,7 +161,9 @@ def __init__(
         password: Text,
         port: Union[int, Text] = 5672,
         queue: Text = "rasa_core_events",
-        loglevel: Union[Text, int] = logging.WARNING,
+        loglevel: Union[Text, int] = os.environ.get(
```

Comment:
not in scope of this PR, but should we add a utils method, also to be used elsewhere? something like 

```python
loglevel = rasa_utils.log_level_from_environment()
```

Reply:
mhm, but that would only replace the one line?